packages/sw-cacheable-response/test/automated-test-suite.js

```diff
@@ -0,0 +1,69 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+const seleniumAssistant = require('selenium-assistant');
+const swTestingHelpers = require('sw-testing-helpers');
+const testServer = require('../../../utils/test-server.js');
+
+const RETRIES = 3;
+const TIMEOUT = 10 * 1000;
+const packageName = 'sw-cacheable-response';
```

Comment:
Could you make this upper case - sorry I was reading this and thought it was some sort of automated script for each package.